### ISSUE TITLE
Random is not defined

### ISSUE THREAD

**Issue by soulwire** [MantarayAR/meteor-shortid#1]
Hi,

I'm using this with Meteor version `1.2.0.2` and `Random` doesn't seem to be defined...

```
ReferenceError: Random is not defined
    at Object.randomByte (packages/mantarayar_shortid/packages/mantarayar_shortid.js:49:1)
    at Object.encode (packages/mantarayar_shortid/packages/mantarayar_shortid.js:310:1)
    at Object.generate (packages/mantarayar_shortid/packages/mantarayar_shortid.js:376:1)
```

The random package is no longer shipped with Meteor so perhaps it can be defined as a dependency in your package.js?

**Comment by idmontie** [comment]
Most likely. This was written back when Random was shipped with Meteor. I can look at this later tonight.

**Comment by idmontie** [comment]
Forgot to reference this in a commit. Closed via f5623ca3b06a1632647423af498b8bcf09207c26
